### ISSUE TITLE
Missing P/T indicator on creatures drag&dropped from the deck

### ISSUE THREAD

**Issue by ctrlaltca** [Cockatrice/Cockatrice#1882]
The P/T indicator on creatures gets added to the card when it's moved from any zone to the tablezone.
If the card is not revealed and is drag&dropped directly from the deck to the table, the P/T indicator doesn't get correctly added.
This is due to the fact that the card's informations are not known when it's still on the deck face down, so [the tablezone code](https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/tablezone.cpp#L151) that tries to get the P/T value from the card fails.

**Comment by Jinxstands4Jinx** [comment]
I like this Idea but Beware. when tutoring for cards Face down we also do not want the p/w to carry over. If this is not a concern then pay me no heed sorry.

**Comment by leematos** [comment]
@Jinxstands4Jinx can you elaborate? I think you are talking about manifesting cards? This is specifically (@ctraltca correct me if I'm wrong) about dragging a card from the top of the library into play & when it hits play it _won't_ have P/T because as a "face-down card" on the top of the library it doesn't have a P/T.

**Comment by Jinxstands4Jinx** [comment]
I mean like a Demonic tutor fetching a creature. Demonic tutor like many other tutors do NOT reveal a card to an opponent. So what I mean is i do not want my demonic tutor for a Acidic slime to reveal that I grabbed a "unknown" 2/2 when it should not.

**Comment by leematos** [comment]
@Jinxstands4Jinx thanks for clarifying! When you do that, you right click (or shortcut key) to "view library", grab your card, and drag it (or use the menut) to put it your hand, correct? If so, this will not affect that (as per my understanding)

**Comment by Jinxstands4Jinx** [comment]
Yes and no. Some people will put the card Face down on the field for multiple reasons. 1) they are not sure this is the card they want and are thinking still. 2) to clarify that they are not just sliding a card into their hand as an extra step. 3) The tutor allows the grabbing of multiple cards and they are sorting options.  I just wanted to warn the developers about this and make sure a glitch is not created.  And further clarifying here. the method you described is not what "I" do. When I tutor I use the mouse. I open the library and hold shift as I drag a card out and I only close the library when I am sure of my choice. Then after shuffling I slide the card into my hand. I do things this way so that there is no room for error and for my opponent to think me crafty as my name suggests :)

**Comment by leematos** [comment]
http://g.recordit.co/gtDBFpgupp.gif This is a video of the bug, notice that when I drag from top of deck to play face up, it doesn't get P/T, but when it switches zones (stack to battlefield) it get's the P/T box. I think your uses case is safe and thanks for bringing it up!

**Comment by Jinxstands4Jinx** [comment]
No problem wanted to stop Future glitches before they appear :)

**Comment by ctrlaltca** [comment]
@leematos thank you for the awesome gif, a gif id better than a thousand words.

**Comment by tooomm** [comment]
Ahh, ok. You just want to show the indicator on cards that enter face up (and that turn out to be creatures)?

:+1:

**Comment by tooomm** [comment]
P/T is also not correctly set if a creature card is turned/flipped face up on the battlefield. (morph, manifest for example)

**Comment by ZeldaZach** [comment]
This would be very helpful

**Comment by BruebachL** [comment]
Fixed by #6129
